keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -80,9 +80,15 @@ class RetinaNet(keras.Model):
         bounding_box_format: The format of bounding boxes of input dataset. Refer
             [to the keras.io docs](https://keras.io/api/keras_cv/bounding_box/formats/)
             for more details on supported bounding box formats.
-        backbone: optional `keras.Model`. Must implement the `pyramid_level_inputs`
+        backbone: (Optional) `keras.Model`. Must implement the `pyramid_level_inputs`
             property with keys 3, 4, and 5 and layer names as values. If
-            `None`, defaults to `keras_cv.models.ResNet50V2Backbone()`.
+            `None`, defaults to
+            `keras_cv.models.ResNet50Backbone.from_preset('imagenet')`.
+        feature_extractor: (Optional) `keras.Model`.  A feature extractor takes
```

Comment:
We just cut this in #1454 to simplify the code. Why are we reintroducing?

My best summary of the discussion is that no one could provide a compelling example of why a custom feature extractor would be needed.